db/migrate/20230120234349_creates_authors.rb

```diff
@@ -0,0 +1,12 @@
+class CreatesAuthors < ActiveRecord::Migration[7.0]
+  def change
+    create_table :authors do |t|
```

Comment:
- [ ] Let's follow the ERD diagram on table and column names, this table should be named users.